src/icons/file-icon/FileIcon.js.flow

```diff
@@ -2,6 +2,7 @@
 import * as React from 'react';
 
 import IconFileAudio from '../../icon/content/FileAudio32';
+import IconFileCanvas from '../../icon/content/FileCanvas32';
```

Comment:
nit: the usages of the icons are sorted alphabetically in this file and `src/icons/file-icon/FileIcon.tsx`.